myhoard/controller.py

```diff
@@ -1389,6 +1389,11 @@ def _send_binlog_stats(self):
         self.stats.gauge_float("myhoard.binlog_upload_age_min", min(binlog_upload_ages))
         self.stats.gauge_float("myhoard.binlog_upload_age_max", max(binlog_upload_ages))
 
+        # Send additional info about how long 'complete' binlog files have been waiting to be uploaded
+        binlog_upload_delays = [backup_stream.binlog_upload_delay for backup_stream in backup_streams]
+        self.stats.gauge_float("myhoard.binlog_upload_delay_min", min(binlog_upload_delays))
```

Comment:
I'm unsure about the usefulness about the minimum upload delay. But if you think it's OK, it's fine for me to add, it won't break anything.

Reply:
Yes, I think it is OK. Thanks!